instrumentation/opentelemetry-instrumentation-flask/pyproject.toml

```diff
@@ -35,8 +35,8 @@ dependencies = [
 
 [project.optional-dependencies]
 instruments = [
-  "flask >= 1.0, < 3.0",
-  "werkzeug < 3.0.0"
+  "werkzeug < 3.0.0",
```

Comment:
@ocelotl 
Shouldn't we remove the werkzeug version range from this part?

Reply:
Yes, I actually need to review all instrumentations and its dependencies. I'll do that in a separate issue/PR. For the time being, let's not hold this PR back for this thing.